packages/storybook-react/src/stories/Paragraph.stories.tsx

```diff
@@ -1,22 +1,18 @@
 import readme from '@nl-design-system-unstable/documentation/componenten/_paragraph.md?raw';
 import { Meta, StoryObj } from '@storybook/react';
-import { Paragraph } from '@utrecht/component-library-react/dist/css-module/index';
+import { Paragraph } from '@utrecht/component-library-react';
 import tokensDefinition from '@utrecht/components/paragraph/tokens.json';
 import tokens from '@utrecht/design-tokens/dist/index.json';
-import React from 'react';
-import { designTokenStory } from './util';
 
-const meta = {
+export const Default: Meta<typeof Paragraph> = {
   title: 'React Component/Paragraph',
-  id: 'react-paragraph',
   component: Paragraph,
   args: {
     lead: false,
     small: false,
     children:
       'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum',
   },
-  tags: ['autodocs'],
```

Comment:
Deze andere blogpost schrijft dat autodocs opt-in is met deze tag:
https://storybook.js.org/blog/storybook-7-docs/

> **Autodocs is now opt-in**
> Another important change with autodocs is that you have to make the choice of enabling docs page for a component. In Storybook 7, you must add the autodocs tag to your component to generate its autodocs page. This is not enabled by default.
> 
> ```export default {
>   component: Button,
>   tags: ['autodocs'], // enable automatic documentation page
> };
> ```

Is 'ie toch niet nodig? Of is het niet nodig omdat we globally dit aan hebben gezet in `main.js`?
Dan heb ik denk ik toch liever dat we handmatig overal tags hebben, zodat we de vrijheid krijgen om die globally te verwijderen en andere soorten stories te hebben.
https://github.com/nl-design-system/utrecht/blob/main/packages/storybook-react/config/main.js#L37

Reply:
ik zag het idd, maar de docs worden ook zonder de tag ingeladen
<img width="1027" alt="Screenshot 2023-03-02 at 10 25 44" src="https://user-images.githubusercontent.com/125254445/222387378-5afce0da-2603-45ee-8d2d-c9fcb25b559d.png">